bpf/ut/ip_options_test.go

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2019-2020 Tigera, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ut_test
+
+import (
+	"testing"
+
+	. "github.com/onsi/gomega"
+)
+
+func TestIPOptionsFromToWEP(t *testing.T) {
+	RegisterTestingT(t)
+
+	iphdr := *ipv4Default
+	iphdr.IHL = 6
+
+	_, _, _, _, pktBytes, err := testPacket(nil, &iphdr, nil, nil)
+	Expect(err).NotTo(HaveOccurred())
+
+	runBpfTest(t, "calico_from_workload_ep", rulesDefaultAllow, func(bpfrun bpfProgRunFn) {
```

Comment:
I think we should run the test for all from/to wep/hep combinations and if my comment above on the HEP is correct, we should also have a test that would not drop a packet to HEP with the host ip and should not create any CT entry.